src/sentry/templates/sentry/emails/transactions.html

```diff
@@ -46,5 +46,33 @@
             {% endfor %}
         </tr>
         {% endif %}
+        {% if span_evidence_key_value %}
+        {% for item in span_evidence_key_value %}
+            {% if item.is_multi_value %}
+            <tr>
+                <th>{% trans item.key %}</th>
+                <td>
+                    <code>{{ item.value.0|truncatechars:100 }}</code>
+                </td>
+            </tr>
+            {% for val in item.value|slice:"1:" %}
+            <tr>
+                <th></th>
+                <td>
+                    <code>{{ val|truncatechars:100 }}</code>
+                </td>
+            </tr>
+            {% endfor %}
+
+            {% else %}
+            <tr>
+                <th>{% trans item.key %}</th>
+                <td>
+                    <code>{{ item.value|truncatechars:100 }}</code>
+                </td>
+            </tr>
```

Comment:
This would be really good to pull out into a partial, it's all over the place in this file